Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1913,6 +1929,13 @@ static void loadPrimitiveGameThreadPart(
   pMesh->SetCustomDepthStencilValue(
       pGltf->CustomDepthParameters.CustomDepthStencilValue);
 
+  static constexpr std::array<double, 3> emissiveFactor = {1.0, 1.0, 1.0};
+  if (loadResult.pMaterial && std::equal(
+                                  loadResult.pMaterial->emissiveFactor.begin(),
+                                  loadResult.pMaterial->emissiveFactor.end(),
+                                  emissiveFactor.begin())) {
+    pMesh->bCastDynamicShadow = false;
```

Comment:
Are you sure about this? It's possible for a (normal, not unlit) material to both have an emissive component, _and_ the user expects it to cast shadows like normal, right?